tokio/src/runtime/scheduler/multi_thread/queue.rs

```diff
@@ -501,6 +569,13 @@ impl<T> Drop for Local<T> {
 }
 
 impl<T> Inner<T> {
+    fn capacity(&self) -> usize {
+        let (steal, _) = unpack(self.head.load(Acquire));
+        let tail = self.tail.load(Acquire);
+
+        LOCAL_QUEUE_CAPACITY - (tail.wrapping_sub(steal) as usize)
+    }
```

Comment:
This is the amount of remaining space? I don't think we should call that the _capacity_. How about `remaining_slots`?

Reply:
Done